src/crystal/system/unix/file_info.cr

```diff
@@ -54,7 +54,7 @@ struct Crystal::System::FileInfo < ::File::Info
     @stat.st_gid.to_u32
   end
 
-  def ==(other : ::File::Info) : Bool
+  def same_file?(other : ::File::Info) : Bool
```

Comment:
Can we add a `File#==` as well, which does info on both file instances and then `same_file?`?

Reply:
I don't think the semantic of `File#==` to compare files is accurate. The file has more state that the file description.

Reply:
It shouldn't do. But I guess this is fine for now...